python/paddle/v2/trainer.py

```diff
@@ -161,14 +161,14 @@ def train(self, reader, num_passes=1, event_handler=None, feeding=None):
                     self.__parameter_updater__.update(each_param)
                 cost_sum = out_args.sum()
                 cost = cost_sum / len(data_batch)
-                self.__parameter_updater__.finishBatch(cost)
```

Comment:
Just a little confusing, why we move the `batch_evaluator` to the `event_handler`?